ncpi_model_forge/ingest_plugin/ncpi_patient.py

```diff
@@ -0,0 +1,168 @@
+"""
+Builds FHIR Patient resources (https://www.hl7.org/fhir/patient.html) from rows
+of tabular participant data.
+"""
+
+# https://hl7.org/fhir/us/core/ValueSet-omb-ethnicity-category.html
+omb_ethnicity_category = {
+    "Hispanic or Latino": {
+        "url": "ombCategory",
+        "valueCoding": {
+            "system": "urn:oid:2.16.840.1.113883.6.238",
+            "code": "2135-2",
+            "display": "Hispanic or Latino",
+        },
+    },
+    "Not Hispanic or Latino": {
+        "url": "ombCategory",
+        "valueCoding": {
+            "system": "urn:oid:2.16.840.1.113883.6.238",
+            "code": "2186-5",
+            "display": "Not Hispanic or Latino",
+        },
+    },
+}
+
+# https://hl7.org/fhir/us/core/ValueSet-omb-race-category.html
+omb_race_category = {
+    "American Indian or Alaska Native": {
+        "url": "ombCategory",
+        "valueCoding": {
+            "system": "urn:oid:2.16.840.1.113883.6.238",
+            "code": "1002-5",
+            "display": "American Indian or Alaska Native",
+        },
+    },
+    "Asian": {
+        "url": "ombCategory",
+        "valueCoding": {
+            "system": "urn:oid:2.16.840.1.113883.6.238",
+            "code": "2028-9",
+            "display": "Asian",
+        },
+    },
+    "Black or African American": {
+        "url": "ombCategory",
+        "valueCoding": {
+            "system": "urn:oid:2.16.840.1.113883.6.238",
+            "code": "2054-5",
+            "display": "Black or African American",
+        },
+    },
+    "Native Hawaiian or Other Pacific Islander": {
+        "url": "ombCategory",
+        "valueCoding": {
+            "system": "urn:oid:2.16.840.1.113883.6.238",
+            "code": "2076-8",
+            "display": "Native Hawaiian or Other Pacific Islander",
+        },
+    },
+    "White": {
+        "url": "ombCategory",
+        "valueCoding": {
+            "system": "urn:oid:2.16.840.1.113883.6.238",
+            "code": "2106-3",
+            "display": "White",
+        },
+    },
+}
+
+species_dict = {
```

Comment:
I don't know if we should include any of the species stuff yet because the species extension isn't merged into master yet